lxd/device/nic_routed.go

```diff
@@ -205,6 +208,69 @@ func (d *nicRouted) validateEnvironment() error {
 	return nil
 }
 
+func (d *nicRouted) checkIPAvailability(parent string) error {
```

Comment:
Feels like we should run the IP tests concurrently to avoid longer start up delays when using multiple ips